src/modules/groupAccess.ts

```diff
@@ -1,11 +1,36 @@
+import { prisma } from '../utils/prisma';
 import { AccessCheckResult, allowed, notAllowed } from '../utils/access';
 import { SessionUser } from '../utils/auth';
 
 import { tr } from './modules.i18n';
 
+const checkGroupAdmin = async (groupId: string, userId: string) => {
+    const admins = await prisma.$queryRaw<Array<{ count: number }>>`
+    SELECT count(*) FROM public."GroupAdmin"
```

Comment:
Add one more level of indentation for the sql block

Reply:
done

Reply:
indentation, not the empty line